searx/static/themes/simple/src/js/main/keyboard.js

```diff
@@ -43,6 +43,26 @@ searxng.ready(function () {
     }
   });
 
+  searxng.on('.image-detail-open', 'keydown', function (e) {
```

Comment:
Currently `Ctrl-Left`, `Alt-Left` etc... works too.

Can you add something like this:

```js
if (e.ctrlKey || e.altKey || e.shiftKey || e.metaKey) {
  return
}
```

to ignore the event when Ctrl/Alt/Shift/Meta keys are pressed ?

Reply:
Committed, are there anything else I can do?